### PR TITLE
some fixes and changes for simpler testing

### PR DESCRIPTION
**Pull request by lychanl**

### CODE REVIEW

File: src/simulated_annealing.py [hunk 1]
```diff
@@ -38,9 +38,12 @@ def _next_iteration(self):
         adding_probability = len(self._points_set.points_to_add) / (len(self._points_set.points_to_add)
                                                                     + len(self._points_set.points_to_remove))
 
-        if random.random() <= adding_probability:
+        rand = random.random()
+        print(rand, len(self._points_set.points_to_add) + len(self._points_set.points_to_remove), adding_probability)
+        if rand <= adding_probability:
             # select point to add
             point = random.sample(self._points_set.points_to_add, 1)[0]
+            print(hash(point), point, sep=' ')
```
Comment (Delebrith) [inline src/simulated_annealing.py]: do you need this log?

File: src/simulated_annealing.py [hunk 2]
```diff
@@ -38,9 +38,12 @@ def _next_iteration(self):
         adding_probability = len(self._points_set.points_to_add) / (len(self._points_set.points_to_add)
                                                                     + len(self._points_set.points_to_remove))
 
-        if random.random() <= adding_probability:
+        rand = random.random()
+        print(rand, len(self._points_set.points_to_add) + len(self._points_set.points_to_remove), adding_probability)
```
Comment (Delebrith) [inline src/simulated_annealing.py]: Is that necessary?

File: src/simulated_annealing.py [hunk 3]
```diff
@@ -52,6 +55,7 @@ def _next_iteration(self):
         else:
             # select point to remove
             point = random.sample(self._points_set.points_to_remove, 1)[0]
+            print(hash(point), point, sep=' ')
```
Comment (Delebrith) [inline src/simulated_annealing.py]: Same as above

**Review by Delebrith — APPROVED** [review]

File: src/simulated_annealing.py (hunk 1 above)
Comment (lychanl) [reply src/simulated_annealing.py]: no, it will be removed

File: src/simulated_annealing.py (hunk 2 above)
Comment (lychanl) [reply src/simulated_annealing.py]: no, will change

File: src/simulated_annealing.py (hunk 3 above)
Comment (lychanl) [reply src/simulated_annealing.py]: same as above

**Review by Delebrith — APPROVED** [review]

**Pull request merged**